static/src/javascripts/projects/common/modules/search-tool.js

```diff
@@ -68,11 +68,21 @@ define([
                 // Send data to whoever is listening
                 mediator.emit('autocomplete:fetch', data);
                 this.setInputValue();
+                this.track(data.city);
 
                 // Clear all after timeout because of the tracking we can't remove everything straight away
                 setTimeout(this.destroy.bind(this), 50);
             },
 
+            track: function (city) {
+                s.events = 'event100';
+                s.prop22 = city;
+                s.eVar22 = city;
+                s.linkTrackVars = 'prop22,eVar22';
+                s.linkTrackEvents = 'event100';
+                s.tl(this, 'o', 'weather location set by user');
```

Comment:
Are you sure that first parameter is correct? I'm pretty sure it should be `true` like: https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/analytics/omniture.js#L111